client/src/app/components/environments/environment-list/environment-list-view.component.ts

```diff
@@ -23,6 +23,8 @@ export class EnvironmentListViewComponent {
 
   editable = input<boolean | undefined>();
   deployable = input<boolean | undefined>();
+  hasUnlockPermissions = input<boolean | undefined>();
```

Comment:
tip: you don't have to specify undefined here, it is automatically added to the type when you don't add a default value or use input.required()